src/common.h

```diff
@@ -1,17 +1,20 @@
-
 #ifndef RGRPC_H
 #define RGRPC_H
 
+#include <Rcpp.h>
+void debugLog(std::string message);
```

Comment:
Where was this used?

Reply:
ohh, this was left over ... I had a `common.cpp` file provdigin `debugLog` that read a `debug` option from the R pkg's namespace, but got rid of it after all (although I really liked the design) .. sorry, this should be removed